modules/mod_login/tmpl/default.php

```diff
@@ -91,8 +91,8 @@
 		<?php if (PluginHelper::isEnabled('system', 'remember')) : ?>
 			<div class="mod-login__remember form-group">
 				<div id="form-login-remember-<?php echo $module->id; ?>" class="form-check">
-					<label class="form-check-label">
-						<input type="checkbox" name="remember" class="form-check-input" value="yes">
+					<label for="form-check-label">
+						<input type="checkbox" name="remember" id="form-check-label" value="yes">
```

Comment:
```suggestion
					<label class="form-check-label">
						<input type="checkbox" name="remember" class="form-check-input" value="yes">
```

I'm reverting this because it's wrong.

The label doesn't need to have a for if the input field is within the label, also the classes are for styling and the id would be input and not label.